ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -3,12 +3,12 @@
 The aim of this API is to expose groups of similar Agents evolving in Unity
```

Comment:
It seems like there is still a lot of discussion about "groups" here. If the concept no longer exists in the API, should we still be talking about it?